README.md

```diff
@@ -50,7 +50,7 @@ while n < data.len() {
     let res = inflater.update(&data[n..]);
     if let Ok((num_bytes_read, result)) = res {
         n += num_bytes_read;
-        out.extend(result);
+        out.extend(result.iter().map(|v| *v));
```

Comment:
@eddyb yep, `.iter().cloned()` works as well. I pushed 5ca5c77 if you want to check it out